src/utils.js

```diff
@@ -107,10 +107,6 @@ export const createCorrectInternalRoute = (checkPath) => {
     basePath = basePath.slice(0, -1);
   }
 
-  if (!checkPath.startsWith(basePath)) {
-    return `${basePath}${checkPath}`;
-  }
-
```

Comment:
If you remove this, then `createCorrectInternalRoute` becomes a NOOP.  If this is intentional, then the whole function should be removed.

Reply:
Reverted this function to the unchanged version, now it is used only for [EditorContainer](https://github.com/openedx/frontend-app-authoring/blob/master/src/editors/EditorContainer.tsx#L51) component